contrib/tkn-image/README.md

```diff
@@ -0,0 +1,20 @@
+# tkn oci image
+
+This directory contains the `Dockerfile` used to build the official
+`tkn` image. It is a multi-stage `Dockerfile` that defines 3 targets:
+
+- `tkn-alpine` for a alpine-based image
+  ```bash
+  docker build --file=contrib/tkn-image/Dockerfile \
+      --target=tkn-apline tkn:alpine .
```

Comment:
`alpine` is spelled wrong in the target. Also, I believe the there is a `--tag` flag missing before `tkn:alpine`. This tag is missing in the other two commands in this README as well.

Reply:
omfg :man_facepalming: I think I was tired :joy_cat:

Reply:
fixed :angel: